build-tools/android-toolchain/android-toolchain.targets

```diff
@@ -106,10 +106,13 @@
   </Target>
   <Target Name="_SetMxeToolchainMakefileTimeToLastCommitTimestamp"
       Condition=" '$(NeedMxe)' == 'true' And ($(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:')) Or $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win32:')))">
-    <Exec
-        Command="touch -m -t `git log -1 --format=%25cd --date=format-local:%25Y%25m%25d%25H%25M.%25S` Makefile"
-        WorkingDirectory="..\..\external\mxe"
-    />
+    <GitCommitTime
+      WorkingDirectory="..\..\external\mxe"
```

Comment:
Our indentation convention for XML is...different. :-)

Attributes should be one per line -- as you've done -- and attributes should be indented one "tab stop" (two spaces) *further than* nested elements. This is to make attributes and elements easier to distinguish visually:

```xml
<GitCommitTime
    WorkingDirector="..\..\external\mxe"
    ToolPath="$(GitToolPath)"
    ToolExe="$(GitToolExe)">
  <Output  TaskParameter="Time" PropertyName="_MxeCommitTime" />
</GitCommitTime>
```